paasta_tools/mesos_tools.py

```diff
@@ -47,6 +49,7 @@
 
 
 DEFAULT_MESOS_CLI_CONFIG_LOCATION = "/nail/etc/mesos-cli.json"
+DEFAULT_SERVICES_LOCATION = "/nail/etc/services/services.yaml"
```

Comment:
This isn't meant to be a plugin, just an import. If I made it to be the latter, paasta tools would have to be given the executor to initialize so that it could pass in the address. I'll make it configurable, but given that we are going paasta is going to initialize taskproc anyway, it will either have to have the info elsewhere, or allow the script to take options.